crates/sui-bridge/src/config.rs

```diff
@@ -226,7 +233,8 @@ pub struct BridgeClientConfig {
     pub eth_bridge_contracts: Vec<EthAddress>,
     pub sui_bridge_modules: Vec<Identifier>,
     pub eth_bridge_contracts_start_block_override: BTreeMap<EthAddress, u64>,
-    pub sui_bridge_modules_start_tx_override: BTreeMap<Identifier, TransactionDigest>,
+    /// The EventID needs to be valid, namely it exists and matches the filter. Otherwise, it will miss one event.
```

Comment:
How about:
```suggestion
    /// The EventID needs to be valid, namely it exists and matches the filter. Otherwise, it will miss one event because of fullnode query semantics.
```